advanced/mathematical_optimization/examples/helper/compare_optimizers.py

```diff
@@ -15,30 +15,31 @@
     rosenbrock_prime, rosenbrock_hessian, LoggingFunction, \
     CountingFunction
 
-def my_partial(function, **kwargs):
+def my_partial(**kwargs):
+    function = optimize.minimize
     f = functools.partial(function, **kwargs)
     functools.update_wrapper(f, function)
     return f
 
 methods = {
-    'Nelder-mead':          my_partial(optimize.fmin,
-                                        ftol=1e-12, maxiter=5e3,
-                                        xtol=1e-7, maxfun=1e6),
-    'Powell':               my_partial(optimize.fmin_powell,
-                                        ftol=1e-9, maxiter=5e3,
-                                        maxfun=1e7),
-    'BFGS':                 my_partial(optimize.fmin_bfgs,
-                                        gtol=1e-9, maxiter=5e3),
-    'Newton':               my_partial(optimize.fmin_ncg,
-                                        avextol=1e-7, maxiter=5e3),
-    'Conjugate gradient':   my_partial(optimize.fmin_cg,
-                                        gtol=1e-7, maxiter=5e3),
-    'L-BFGS':               my_partial(optimize.fmin_l_bfgs_b,
-                                        approx_grad=1, factr=10.0,
-                                        pgtol=1e-8, maxfun=1e7),
-    "L-BFGS w f'":          my_partial(optimize.fmin_l_bfgs_b,
-                                        factr=10.0,
-                                        pgtol=1e-8, maxfun=1e7),
+    'Nelder-mead':          my_partial(method="Nelder-Mead", options = {
```

Comment:
PEP8 tells us that there should no space around the equal sign after "options" (here and below)